File: sandbox/node/src/AccountCreateDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.AccountApi();

const accountCreateRequest = new models.AccountCreateRequest();
accountCreateRequest.emailAddress = "newuser@dropboxsign.com";

apiCaller.accountCreate(
    accountCreateRequest,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Account#accountCreate:");
  console.log(error.body);
});

File: sandbox/ruby/src/AccountCreateDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

account_create_request = Dropbox::Sign::AccountCreateRequest.new
account_create_request.email_address = "newuser@dropboxsign.com"

begin
    response = Dropbox::Sign::AccountApi.new.account_create(
        account_create_request,
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling Account#account_create: #{e}"
end
